install.md

```diff
@@ -195,6 +195,96 @@ echo 'deb [signed-by=/usr/share/keyrings/projectatomic-ppa.gpg] http://ppa.launc
 apt update
 apt -y install -t stretch-backports libostree-dev golang
 apt -y install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libseccomp-dev libselinux1-dev skopeo-containers go-md2man
-``` 
+```
 
 The build steps on Debian are otherwise the same as Ubuntu, above.
+
+## Configuration files
+
+### [registries.conf](https://github.com/containers/image/blob/master/docs/registries.conf.5.md)
+
+`/etc/containers/registries.conf`
+
```

Comment:
I commented in the prior PR that it would be nice to have an intro paragraph here listing the files in this section noting the ones that are required for basic Buildah operations.  Not sure if that got to you or not.